root_locals.tf

```diff
@@ -124,6 +124,6 @@ locals {
 
   //feature access blocks
   block_assign_file_references = local.environment == "intg" ? false : true
-  block_validation_library     = local.environment == "intg" ? false : true
+  block_validation_library     = (local.environment == "intg" || local.environment == "staging") ? false : true
```

Comment:
Not that its that important but can change this to: `block_validation_library = local.environment == "prod" ? true : false`

Just a little bit simpler.

Reply:
Yeah, thanks.